basket/BABY_LOUIE/gulpfile.babel.js

```diff
@@ -28,7 +32,11 @@ function loadConfig() {
 
 // Build the "dist" folder by running all of the below tasks
 gulp.task('build',
- gulp.series(clean, gulp.parallel(sass, javascript, images)));
+<<<<<<< HEAD
```

Comment:
@cloudsociety What's going on here?  Github says no conflicts.  Did you commit conflict markers?